code/API_definitions/geofencing.yaml

```diff
@@ -441,26 +441,33 @@ components:
       description: Subscription detail
       properties:
         device:
-          $ref: '#/components/schemas/Device'
+          $ref: "#/components/schemas/Device"
         area:
-          $ref: '#/components/schemas/Area'
+          $ref: "#/components/schemas/Area"
         type:
-          $ref: "#/components/schemas/Type"
+          $ref: "#/components/schemas/SubscriptionCreationEventType"
       required:
         - device
         - area
         - type
 
-    Type:
+    SubscriptionCreationEventType:
+      type: string
       description: |
         area-entered - Event triggered when the device enters the given area
         area-left - Event triggered when the device leaves the given area
-        subscription-ends - Event triggered when the subscription ends
-      type: string
       enum:
         - org.camaraproject.geofencing.v0.area-entered
         - org.camaraproject.geofencing.v0.area-left
-        - org.camaraproject.geofencing.v0.subscription-ends
+
+    SubscriptionEventType:
+      allOf:
```

Comment:
We cannot "extend" an enum this way, as allOf requires compliance to both schemas, and any of the specific values cannot comply with both schemas at the same time. We have to define a new enum with the 3 values, same model as in https://github.com/camaraproject/DeviceStatus/blob/main/code/API_definitions/device-status.yaml#L568